week-2/L-array-methods-2/exercise.js

```diff
@@ -15,11 +15,11 @@ var everyone = [
   "Swathi"
 ];
 
-var firstFive; // complete this statement
-var lastFive; // complete this statement
+var firstFive = everyone.slice(0, 5); // complete this statement
+var lastFive = everyone.slice(everyone.length - 5, everyone.length + 1); // complete this statement
```

Comment:
You don't need the `+1` in `everyone.length + 1` here. The last element in an array `a` is `a[ a.length - 1]`. So `a[a.length]` is already past the end of the array.